docs/product-assembly_dmn-authoring-guide/src/main/asciidoc/main.adoc

```diff
@@ -9,7 +9,7 @@ include::product-shared-docs/document-attributes.adoc[]
 include::product-shared-docs/author-group.adoc[]
 
 // Purpose statement for the assembly
-Decision Modeling Notation (DMN) is an open standard created by the Object Management Group (OMG). With DMN, organizations model their decision processes using a graphical language to build a diagram of decisions and requirements called a Decision Requirements Graph. This graph traces business decisions from start to finish, with each decision point drawing logic from custom tables known as decision tables. 
+Decision Modeling and Notation (DMN) is an open standard created by the Object Management Group (OMG). With DMN, organizations model their decision processes using a graphical language to build a diagram of decisions and requirements called a Decision Requirements Graph. This graph traces business decisions from start to finish, with each decision point drawing logic from DMN model decision elements such as decision tables. 
```

Comment:
It should be Decision Model and Notation, not Modeling. See here [1].  

[1] https://www.omg.org/spec/DMN/About-DMN/